skrub/_table_vectorizer.py

```diff
@@ -146,10 +150,72 @@ def _replace_missing_in_cat_col(ser: pd.Series, value: str = "missing") -> pd.Se
     return ser
 
 
-Transformer = TransformerMixin | Literal["drop", "remainder", "passthrough"]
+def _clone_if_default(transformer, DEFAULT_TRANSFORMER):
+    return clone(transformer) if transformer is DEFAULT_TRANSFORMER else transformer
+
+
+def _clone_during_fit(transformer, remainder, n_jobs):
+    if isinstance(transformer, sklearn.base.TransformerMixin):
+        return _propagate_n_jobs(clone(transformer), n_jobs)
+    elif transformer is None:
```

Comment:
I think we might want to avoid this conversion of `None` to `"passthrough"`:
- it is not super obvious whether `None` should mean "passthrough" or "drop"
- "passthrough" is already a convenient and more explicit way to mean "passthrough"
- if we don't give a special meaning to None now (making it an invalid input ATM) we keep the possibility to give it one later (eg if we decide to revisit our way of specifying the default parameter, for example if one day we decide we want the `TableVectorizer` to pass the `check_estimator` tests)

Reply:
Ok, sounds good!